Client/core/Core.cpp

```diff
@@ -30,6 +30,12 @@ int WINAPI DllMain(HINSTANCE hModule, DWORD dwReason, PVOID pvNothing)
         WriteDebugEvent(SString("DLL_PROCESS_ATTACH %08x", pvNothing));
         if (IsGTAProcess())
         {
+            if (AllocConsole())
+            {
+                freopen("CONIN$", "r", stdin);
+                freopen("CONOUT$", "w", stdout);
+                freopen("CONOUT$", "w", stderr);
+            }
```

Comment:
This is really useful we really need to just have this run in master when we're in Debug mode

Reply:
yes, and it's also much faster than MTA's GUI console.